### PR TITLE
:sparkles: Add meta-cassandra

### PR DESCRIPTION
**Pull request by rinx**
Co-authored-by: autopp <autopp.inc@gmail.com>

### CODE REVIEW

**Comment by rinx** [comment]
Remained tasks:
- [ ] `ClusterConfig` struct fields should be fully configurable [[link]](https://github.com/gocql/gocql/blob/ae2f7fc85f32248f9341a280ccdad16b44581f36/cluster.go#L30)

File: internal/config/cassandra.go [hunk 1]
```diff
@@ -18,12 +18,57 @@
...
+	PageSize               int      `json:"page_size" yaml:"page_size"`
+	DefaultTimestamp       bool     `json:"default_timestamp" yaml:"default_timestamp"`
+	MaxWaitSchemaAgreement string   `json:"max_wait_schema_agreement" yaml:"max_wait_schema_agreement"`
+	ReconnectInterval      string   `json:"reconnect_interval" yaml:"reconnect_interval"`
+	// ConvictionPolicy:       &SimpleConvictionPolicy{},
+	ReconnectionPolicy    *ReconnectionPolicy `json:"reconnection_policy" yaml:"reconnection_policy"`
+	WriteCoalesceWaitTime string              `json:"write_coalesce_wait_time" yaml:"write_coalesce_wait_time"`
+
+	Keyspace string `json:"keyspace" yaml:"keyspace"`
+	KVTable  string `json:"kv_table" yaml:"kv_table"`
```
Comment (kpango) [inline internal/config/cassandra.go]: table? keyspace?

Comment (rinx) [reply internal/config/cassandra.go]: GoCQL cannot connect to multiple keyspaces simultaneously. Then, we decided to hold multiple tables in single keyspace. :cry:

Comment (kpango) [reply internal/config/cassandra.go]: omg

**Comment by kpango** [comment]
LGTM

**Pull request merged**